.github/workflows/build_all.yml

```diff
@@ -69,8 +69,8 @@ jobs:
       - name: Install packages
         run: |
           brew update
-          brew install ninja python
-          python3 -m pip install git+https://github.com/mesonbuild/meson
+          brew install ninja
+          python3 -m pip install --pre meson
```

Comment:
Absolutely not!!!

The purpose of this was extremely clear. We want to test from meson git master, it's the *entire* point of the build_all check.

Reply:
Should be one or the other globally I think.